src/main.rs

```diff
@@ -26,6 +28,7 @@ fn main() {
     opts.optopt("R", "", "changes the git remote used (default is origin)", "REMOTE");
     opts.optopt("b", "", "changes the base for merged branches (default is master)", "BRANCH");
     opts.optflag("h", "help", "print this help menu");
+    opts.optflag("", "version", "print the version");
```

Comment:
I think the common way to go if you add verbose as well is to use `-v` for `verbose` and `-V` for version.

That said, if we don't plan to add verbose, I vote for the lower case `-v`.